File: app/src/main/java/com/intelegain/agora/model/Employee_contacts.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by maninder on 27/7/17.
 */

public class Employee_contacts {

    public String strUrl, strEmpName, strEmpDesingation, strMail, strPhoneNumber;
    public boolean isSwiped;


    @SerializedName("empid")
    @Expose
    public Integer empid;
    @SerializedName("ProfileImageURL")
    @Expose
    public String ProfileImageURL;
    @SerializedName("empName")
    @Expose
    public String empName;
    @SerializedName("empAddress")
    @Expose
    public String empAddress;
    @SerializedName("empGender")
    @Expose
    public String empGender;
    @SerializedName("empContact")
    @Expose
    public String empContact;
    @SerializedName("empJoiningDate")
    @Expose
    public String empJoiningDate;
    @SerializedName("empLeavingDate")
    @Expose
    public String empLeavingDate;
    @SerializedName("empProbationPeriod")
    @Expose
    public Integer empProbationPeriod;
    @SerializedName("empNotes")
    @Expose
    public String empNotes;
    @SerializedName("empEmail")
    @Expose
    public String empEmail;
    @SerializedName("empTester")
    @Expose
    public Boolean empTester;
    @SerializedName("empAccountNo")
    @Expose
    public String empAccountNo;
    @SerializedName("EmpPAN")
    @Expose
    public String empPAN;
    @SerializedName("EmpUAN")
    @Expose
    public String empUAN;
    @SerializedName("EmpEPF")
    @Expose
    public String empEPF;
    @SerializedName("empBDate")
    @Expose
    public String empBDate;
    @SerializedName("empADate")
    @Expose
    public String empADate;
    @SerializedName("empPrevEmployer")
    @Expose
    public String empPrevEmployer;
    @SerializedName("empExperince")
    @Expose
    public Integer empExperince;
    @SerializedName("IsSuperAdmin")
    @Expose
    public Boolean isSuperAdmin;
    @SerializedName("IsAccountAdmin")
    @Expose
    public Boolean isAccountAdmin;
    @SerializedName("IsPayrollAdmin")
    @Expose
    public Boolean isPayrollAdmin;
    @SerializedName("IsPM")
    @Expose
    public Boolean isPM;
    @SerializedName("IsProjectReport")
    @Expose
    public Boolean isProjectReport;
    @SerializedName("IsProjectStatus")
    @Expose
    public Boolean isProjectStatus;
    @SerializedName("IsLeaveAdmin")
    @Expose
    public Boolean isLeaveAdmin;
    @SerializedName("IsActive")
    @Expose
    public Boolean isActive;
    @SerializedName("LocationFKID")
    @Expose
    public Integer locationFKID;
    @SerializedName("skillid")
    @Expose
    public Integer skillid;
    @SerializedName("InsertedOn")
    @Expose
    public String insertedOn;
    @SerializedName("InsertedBy")
    @Expose
    public Integer insertedBy;
    @SerializedName("InsertedIP")
    @Expose
    public String insertedIP;
    @SerializedName("ModifiedOn")
    @Expose
    public String modifiedOn;
    @SerializedName("ModifiedBy")
    @Expose
    public Integer modifiedBy;
    @SerializedName("ModifiedIP")
    @Expose
    public String modifiedIP;
    @SerializedName("IsTester")
    @Expose
    public Boolean isTester;
    @SerializedName("empPassword")
    @Expose
    public String empPassword;
    @SerializedName("AnnualCTC")
    @Expose
    public Integer annualCTC;
    @SerializedName("CTC")
    @Expose
    public Integer cTC;
    @SerializedName("Gross")
    @Expose
    public Integer gross;
    @SerializedName("Net")
    @Expose
    public Integer net;
    @SerializedName("Resume")
    @Expose
    public String resume;
    @SerializedName("Qualification")
    @Expose
    public String qualification;
    @SerializedName("QualificationId")
    @Expose
    public String qualificationId;
    @SerializedName("SecSkills")
    @Expose
    public String secSkills;
    @SerializedName("SecSkillsId")
    @Expose
    public String secSkillsId;
    @SerializedName("PrimarySkill")
    @Expose
    public Integer primarySkill;
    @SerializedName("Designation")
    @Expose
    public String designation;
    @SerializedName("Skill")
    @Expose
    public String skill;
    @SerializedName("Type")
    @Expose
    public String type;
    @SerializedName("Photo")
    @Expose
    public String photo;
    @SerializedName("Mode")
    @Expose
    public String mode;
    @SerializedName("EmpStatus")
    @Expose
    public String empStatus;
    @SerializedName("LeavingStatus")
    @Expose
    public String leavingStatus;
    @SerializedName("ProjectID")
    @Expose
    public Integer projectID;
    @SerializedName("SecurityLevel")
    @Expose
    public Integer securityLevel;
    @SerializedName("PrimarySkillDesc")
    @Expose
    public String primarySkillDesc;
    @SerializedName("Event")
    @Expose
    public String event;
    @SerializedName("ProfileID")
    @Expose
    public Integer profileID;
    @SerializedName("CAddress")
    @Expose
    public String cAddress;
    @SerializedName("HistoryID")
    @Expose
    public Integer historyID;
    @SerializedName("ADUserName")
    @Expose
    public String aDUserName;
    @SerializedName("Flag")
    @Expose
    public String flag;
    @SerializedName("strBirthday")
    @Expose
    public String strBirthday;
}

File: app/src/main/java/com/intelegain/agora/model/Employee_contacts.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by maninder on 27/7/17.
 */
class Employee_contacts {
    var strUrl: String? = null
    var strEmpName: String? = null
    var strEmpDesingation: String? = null
    var strMail: String? = null
    var strPhoneNumber: String? = null
    var isSwiped = false
    @SerializedName("empid")
    @Expose
    var empid: Int? = null
    @SerializedName("ProfileImageURL")
    @Expose
    var ProfileImageURL: String? = null
    @SerializedName("empName")
    @Expose
    var empName: String? = null
    @SerializedName("empAddress")
    @Expose
    var empAddress: String? = null
    @SerializedName("empGender")
    @Expose
    var empGender: String? = null
    @SerializedName("empContact")
    @Expose
    var empContact: String? = null
    @SerializedName("empJoiningDate")
    @Expose
    var empJoiningDate: String? = null
    @SerializedName("empLeavingDate")
    @Expose
    var empLeavingDate: String? = null
    @SerializedName("empProbationPeriod")
    @Expose
    var empProbationPeriod: Int? = null
    @SerializedName("empNotes")
    @Expose
    var empNotes: String? = null
    @SerializedName("empEmail")
    @Expose
    var empEmail: String? = null
    @SerializedName("empTester")
    @Expose
    var empTester: Boolean? = null
    @SerializedName("empAccountNo")
    @Expose
    var empAccountNo: String? = null
    @SerializedName("EmpPAN")
    @Expose
    var empPAN: String? = null
    @SerializedName("EmpUAN")
    @Expose
    var empUAN: String? = null
    @SerializedName("EmpEPF")
    @Expose
    var empEPF: String? = null
    @SerializedName("empBDate")
    @Expose
    var empBDate: String? = null
    @SerializedName("empADate")
    @Expose
    var empADate: String? = null
    @SerializedName("empPrevEmployer")
    @Expose
    var empPrevEmployer: String? = null
    @SerializedName("empExperince")
    @Expose
    var empExperince: Int? = null
    @SerializedName("IsSuperAdmin")
    @Expose
    var isSuperAdmin: Boolean? = null
    @SerializedName("IsAccountAdmin")
    @Expose
    var isAccountAdmin: Boolean? = null
    @SerializedName("IsPayrollAdmin")
    @Expose
    var isPayrollAdmin: Boolean? = null
    @SerializedName("IsPM")
    @Expose
    var isPM: Boolean? = null
    @SerializedName("IsProjectReport")
    @Expose
    var isProjectReport: Boolean? = null
    @SerializedName("IsProjectStatus")
    @Expose
    var isProjectStatus: Boolean? = null
    @SerializedName("IsLeaveAdmin")
    @Expose
    var isLeaveAdmin: Boolean? = null
    @SerializedName("IsActive")
    @Expose
    var isActive: Boolean? = null
    @SerializedName("LocationFKID")
    @Expose
    var locationFKID: Int? = null
    @SerializedName("skillid")
    @Expose
    var skillid: Int? = null
    @SerializedName("InsertedOn")
    @Expose
    var insertedOn: String? = null
    @SerializedName("InsertedBy")
    @Expose
    var insertedBy: Int? = null
    @SerializedName("InsertedIP")
    @Expose
    var insertedIP: String? = null
    @SerializedName("ModifiedOn")
    @Expose
    var modifiedOn: String? = null
    @SerializedName("ModifiedBy")
    @Expose
    var modifiedBy: Int? = null
    @SerializedName("ModifiedIP")
    @Expose
    var modifiedIP: String? = null
    @SerializedName("IsTester")
    @Expose
    var isTester: Boolean? = null
    @SerializedName("empPassword")
    @Expose
    var empPassword: String? = null
    @SerializedName("AnnualCTC")
    @Expose
    var annualCTC: Int? = null
    @SerializedName("CTC")
    @Expose
    var cTC: Int? = null
    @SerializedName("Gross")
    @Expose
    var gross: Int? = null
    @SerializedName("Net")
    @Expose
    var net: Int? = null
    @SerializedName("Resume")
    @Expose
    var resume: String? = null
    @SerializedName("Qualification")
    @Expose
    var qualification: String? = null
    @SerializedName("QualificationId")
    @Expose
    var qualificationId: String? = null
    @SerializedName("SecSkills")
    @Expose
    var secSkills: String? = null
    @SerializedName("SecSkillsId")
    @Expose
    var secSkillsId: String? = null
    @SerializedName("PrimarySkill")
    @Expose
    var primarySkill: Int? = null
    @SerializedName("Designation")
    @Expose
    var designation: String? = null
    @SerializedName("Skill")
    @Expose
    var skill: String? = null
    @SerializedName("Type")
    @Expose
    var type: String? = null
    @SerializedName("Photo")
    @Expose
    var photo: String? = null
    @SerializedName("Mode")
    @Expose
    var mode: String? = null
    @SerializedName("EmpStatus")
    @Expose
    var empStatus: String? = null
    @SerializedName("LeavingStatus")
    @Expose
    var leavingStatus: String? = null
    @SerializedName("ProjectID")
    @Expose
    var projectID: Int? = null
    @SerializedName("SecurityLevel")
    @Expose
    var securityLevel: Int? = null
    @SerializedName("PrimarySkillDesc")
    @Expose
    var primarySkillDesc: String? = null
    @SerializedName("Event")
    @Expose
    var event: String? = null
    @SerializedName("ProfileID")
    @Expose
    var profileID: Int? = null
    @SerializedName("CAddress")
    @Expose
    var cAddress: String? = null
    @SerializedName("HistoryID")
    @Expose
    var historyID: Int? = null
    @SerializedName("ADUserName")
    @Expose
    var aDUserName: String? = null
    @SerializedName("Flag")
    @Expose
    var flag: String? = null
    @SerializedName("strBirthday")
    @Expose
    var strBirthday: String? = null
}